Makefile

```diff
@@ -1,21 +1,30 @@
 all: build
 
 APP_NAME=cluster-monitoring-operator
+BIN=operator
 MAIN_PKG=github.com/openshift/$(APP_NAME)/cmd/operator
 REPO?=quay.io/coreos/$(APP_NAME)
 TAG?=$(shell git rev-parse --short HEAD)
-ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+ENVVAR=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 NAMESPACE=openshift-monitoring
 KUBECONFIG?=$(HOME)/.kube/config
-PKGS   = $(shell go list ./... | grep -v -E '/vendor/|/test|/examples')
-GOOS   = linux
+PKGS=$(shell go list ./... | grep -v -E '/vendor/|/test|/examples')
+GOOS=linux
 VERSION=$(shell cat VERSION | tr -d " \t\n\r")
+SRC=$(shell find . -type f -name '*.go') pkg/manifests/bindata.go
+GOBINDATA_BIN=$(GOPATH)/bin/go-bindata
+GOJSONTOYAML_BIN=$(GOPATH)/bin/gojsontoyaml
+JSONNET_BIN=$(if $(shell which jsonnet),$(shell which jsonnet),$(GOPATH)/bin/jsonnet)
```

Comment:
we need to be able to have jsonnet on travis. here we default to the user's installed jsonnet binary; if nothing is installed, then install go-jsonnet

Reply:
Your PR comment could probably be a comment in the Makefile.